src/families/crypto_org/logic.js

```diff
@@ -0,0 +1,63 @@
+// @flow
+import { CroNetwork, CroSDK, utils } from "@crypto-com/chain-jslib";
+
+export const TESTNET_CURRENCY_ID = "crypto_org_croeseid";
+
+export const FIXED_GAS_PRICE = 0.025;
+export const FIXED_DEFAULT_GAS_LIMIT = 200000;
+
+export const TestnetCroeseid3 = {
+  defaultNodeUrl: "https://testnet-croeseid-3.crypto.org",
+  chainId: "testnet-croeseid-3",
+  addressPrefix: "tcro",
+  validatorAddressPrefix: "tcrocncl",
+  validatorPubKeyPrefix: "tcrocnclconspub",
+  coin: {
+    baseDenom: "basetcro",
+    croDenom: "tcro",
+  },
+  bip44Path: {
+    coinType: 1,
+    account: 0,
+  },
+  rpcUrl: "https://testnet-croeseid-3.crypto.org:26657",
+};
+
+let croSdk = null;
+
+/**
+ * Get CroSdk
+ * @param {boolean} useTestNet
+ */
+export function getCroSdk(useTestNet: boolean) {
```

Comment:
Kind of the same singleton issue as for API here.
If multiple accounts are added to Ledger Live with different networks, it might not work.

Reply:
fixed